videos/messages.py

```diff
@@ -0,0 +1,44 @@
+"""message classes for videos"""
+from types import SimpleNamespace
+
+from mitol.mail.messages import TemplatedMessage
+
+
+class YouTubeUploadSuccessMessage(TemplatedMessage):
+    """Email message for Youtube upload success"""
+
+    name = "YouTube Video Upload Completed"
+    template_name = "mail/youtube_upload_success"
+
+    @staticmethod
+    def get_debug_template_context() -> dict:
+        """Returns the extra context for the email debugger"""
+        return {
+            "site": SimpleNamespace(
```

Comment:
Are `site` and `video` used? Why are they hardcoded? Here and in YouTubeUploadFailureMessage

Reply:
This is just for debugging purposes, there is a url you can go to for previewing the messages: http://localhost:8043/__emaildebugger__/

But I realized that this was missing from `urls.py` and also the new message classes had to be added to `MITOL_MAIL_MESSAGE_CLASSES`, the above url should work now with the latest commit.